node-operators/networks/collator.md

```diff
@@ -11,49 +11,55 @@ description: 学习如何在运行节点后成为Moonbeam网络的收集人
 
 收集人在网络上维护其所参与的平行链。他们运行（所在平行链及中继链的）全节点，并为中继链验证人创建状态转移证明。
 
-用户可以在Moonbase Alpha和Moonriver上启动全节点，并启用`collate`功能作为候选收集人参与生态系统的运行。候选人进入收集人有效集后有资格生产区块。
+用户可以在Moonbeam、Moonriver和Moonbase Alpha上启动全节点，并启用`collate`功能作为候选收集人参与生态系统的运行。候选人进入收集人有效集后有资格生产区块。
 
 Moonbeam使用[Nimbus平行链共识框架](/learn/features/consensus/)，通过一个两步过滤器将候选人分配到收集人有效集，随后再分配到区块生产插槽：
 
- - 平行链质押过滤器根据每个网络中的Token质押量挑选Moonbase Alpha上排名前{{ networks.moonbase.staking.max_candidates }}名候选人和Moonriver上排名前{{ networks.moonriver.staking.max_candidates }}名候选人。这个过滤后的池被称为“精选候选人池”。每一轮这个池中的候选人都会进行更新
+ - 平行链质押过滤器根据每个网络中的Token质押量挑选排名靠前的候选人。每个网络候选人的具体排名数量，请查看本教程的[账户与质押要求](#accounts-and-staking-requirements)部分。这个过滤后的池被称为“精选候选人池”（即收集人有效集）。每一轮这个池中的候选人都会进行更新
  - 固定规模子集过滤法在第一次过滤的基础之上对每个区块生产插槽进行伪随机的子集选择
 
-此教程将带您完成以下步骤：
+本教程将向您展示成为候选收集人的基本要求，如何加入和离开候选收集人池，更改候选收集人绑定数量，以及创建和映射会话密钥。本教程将以Moonbase Alpha为例，操作步骤也同样适用于Moonbeam或Moonriver。
 
- - **[技术要求](#technical-requirements)** —— 展示您必须满足的技术角度标准
- - **[账户与质押要求](#accounts-and-staking-requirements)** —— 如何完成您的帐户设置并绑定Token成为候选人
- - **[生成会话密钥](#generate-session-keys)** —— 解释说明如何生成会话密钥（用于将您的author ID映射到您的H160帐户）
- - **[映射author ID至您的账户](#map-author-id-to-your-account)** —— 概述如何将您的公共会话密钥映射到您的H160帐户（用于接收区块奖励）
-
-## 技术要求 {: #technical-requirements }
+## 技术要求 {: #technical-requirements } 
 
 从技术角度来看，收集人必须满足以下技术要求：
 
  - 必须运行带有收集选项的全节点。可根据[启动全节点教程](/node-operators/networks/run-a-node/overview/)选择收集人的特定代码段
 
-## 账户与质押要求 {: #accounts-and-staking-requirements }
+!!! 注意事项
+
+​    运行一个**收集人**节点对CPU的要求高于上述所提供的要求。为了使您的收集人节点能够跟上高交易吞吐量，具有高时钟速度和单核性能的CPU非常重要，因为区块生产/导入过程几乎完全是单线程的。我们使用以下CPU进行了测试并看到了良好的结果：Intel Xeon E-2386/2388和Ryzen 9 5950x/5900x。由于大多数云服务提供商专注于多线程而非单线程性能，因此建议使用裸机提供商。另外，因为Docker将对性能产生重大影响，因此也不建议在Docker中运行收集人节点。
+
+## 账户与质押要求 {: #and-staking-requirements } 
```

Comment:
this should be `## Accounts and Staking Requirements {: #accounts-and-staking-requirements }`